packages/config/config/devices/0x019b/4512689.json

```diff
@@ -21,9 +21,25 @@
 		"max": "255.255"
 	},
 	"metadata": {
-		"inclusion": "1. Before you start, find the circuit breaker or fuse box. \n2. Once you have found it, open the door and turn off the main power switch. Step \n3. Two Wago 221 Splicing Connectors are provided. Take out one connector. Pull the lever up and insert the white wire. \n4. Push the lever back down. The transparent housing allows you to check if the wire is connected properly. Make sure the wire is tightly held in place before you proceed. \n5. Repeat Step 3 and Step 4 to insert the black wire. Inserting the two wires to the same side (right), as shown below, of the two connectors provides an easier installation in the following steps. \n7. Put the Z-Wave gateway or control panel into Inclusion or Learning mode (please refer to the Z-Wave gateway or control panel manual). \n8. Within 1.5 seconds, press the Function button 3 times. The Smoke Detector will emit a 2-tone beep and the LED will turn on for ~2 seconds. Refer to the operation manual of the Z-Wave gateway or control panel to complete the learn-in process. \n6. Insert the AC wires to the two connectors respectively, as shown below. Use a pair of tweezers to turn the Battery Switch to ON position (left side), the rechargeable battery will begin to charge.",
-		"exclusion": "1. Put the Z-Wave gateway or control panel into Exclusion mode (please refer to the Z-Wave gateway or control panel manual). \n2. Within 1.5 seconds, press the Function button 3 times and the Smoke Detector will be removed from the Z-Wave network",
-		"reset": "(Only use factory reset when network Control Panel/Gateway is missing or inoperable)\n1. Remove the batteries of the Smoke Detector first.  \n2. Press and hold the Function button. While holding the Function button, power on the Smoke Detector by re-inserting the batteries, wait for 20 seconds to factory reset.",
-		"manual": "https://products.z-wavealliance.org/ProductManual/File?folder=&filename=product_documents/3933/HeatitZ-SmokeDetector230V.pdf"
+		"inclusion": "Please refer to your primary controller manual on how to enter add/remove mode. The device may only be added or removed from the network if the primary controller is in add/remove mode, and may now be added to the system via the primary controller",
+		"exclusion": "Please refer to your primary controller manual on how to enter add/remove mode. The device may only be added or removed from the network if the primary controller is in add/remove mode, and may now be added to the system via the primary controller",
+		"reset": "After the calibration process is finished, you may access the add/remove mode by pressing the function button three times within 1.5 seconds. Removing the device will reset it to factory defaults",
+		"manual": "https://media.heatit.com/709"
+	},
+
+	"associations": {
```

Comment:
Metadata should be the last entry - I'll need to update the docs in this regard.